packages/framework/src/Model/Store/OpeningHours/StoreOpeningHoursProvider.php

```diff
@@ -144,10 +144,12 @@ protected function getWeekSettingFromData(array $openingHoursData): array
             $dayOfWeekName = $this->getEnglishDayNameFromDayNumber($openingHourData->dayOfWeek);
 
             foreach ($openingHourData->openingHoursRanges as $openingHoursRange) {
-                $weekSetting[$dayOfWeekName][] = $this->formatOpeningHours(
-                    $openingHoursRange->openingTime,
-                    $openingHoursRange->closingTime,
-                );
+                if ($openingHoursRange !== null) {
```

Comment:
This problem still exists if I fill only one of the times.

Reply:
you have right, fixed 🔨